app/src/Components/contentStepWrapperDonneMetheo/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Button, Container, FormControl, FormLabel, Input, Select, Stack } from '@chakra-ui/react';
+
```

Comment:
into the folder /src/constants/GlobalStyles.ts you can find the GlobalColors object , reuse it instead of creating an other colors definition like below

Reply:
please delete this constant it is not used. I forgot to remove it. this will allow us to save time and I pull the DEV branch. Thanks.

Reply:
Here we can see that you're creating a style, but in the code we can't see where you're using it, so it takes up memory space for nothing. If you don't use it, failing to do as Hapi suggests, you remove it once.

Reply:
Ok